tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = py26,py27,py33,py34,py35,pypy
+
+[testenv]
+deps=-rrequirements.txt
```

Comment:
It is actually in #21.  Technically this should be rebased on #21 (I didn't want to squash them into a single PR)